test/system/211-dbus.bats

```diff
@@ -87,3 +87,351 @@ teardown() {
   # shellcheck disable=SC2154
   assert [ ${#stderr_lines[@]} -eq 0 ]
 }
+
+@test "dbus: session bus inside Arch Linux" {
+  create_distro_container arch latest arch-toolbox-latest
+  expected_response="$(gdbus call \
+                         --session \
+                         --dest org.freedesktop.DBus \
+                         --object-path /org/freedesktop/DBus \
+                         --method org.freedesktop.DBus.Peer.Ping)"
+
+  run --keep-empty-lines --separate-stderr "$TOOLBOX" run --distro arch sh -c 'gdbus call \
+                                                                   --session \
+                                                                   --dest org.freedesktop.DBus \
+                                                                   --object-path /org/freedesktop/DBus \
+                                                                   --method org.freedesktop.DBus.Peer.Ping'
```

Comment:
These commands seem to reoccur regularly. Putting them into a global variable could prevent a potential typo?

Reply:
That makes a lot of sense. If @debarshiray agrees, I can create two global variables, one for the session bus and another for the system bus.

Reply:
Sure.  Feel free to submit a pull request.